### PR TITLE
Convert test-cli-integration into a pytest

### PR DESCRIPTION
**Pull request by wlach**

### CODE REVIEW

File: tests/test_cli.py [hunk 1]
```diff
@@ -1,23 +1,410 @@
...
+            server_a_verify2_output_filename,
+            "--output",
+            server_b_aggregation_path,
+        ],
+    )
+    assert result.exit_code == 0
+
+    # FIXME: should have some kind of output of the expected output?
+    # For this case it looks like we get results like this: "AJOhAKEAoQA=". Is
+    # that correct?
```
Comment (wlach) [inline tests/test_cli.py]: @acmiyaguchi do you have any suggestions for this part? this is what I'm seeing looking at the file, but it seems wrong

File: tests/test_cli.py [hunk 2]
```diff
@@ -1,23 +1,410 @@
...
+            server_a_aggregation_output_filename,
+            "--output",
+            server_b_published_path,
+        ],
+    )
+    assert result.exit_code == 0
+
+    # FIXME: should have some kind of output of the expected output?
+    # For this case it looks like we get results like this: [0,0,0]. Is
+    # that correct?
```
Comment (wlach) [inline tests/test_cli.py]: @acmiyaguchi same as above: this is what I'm seeing looking at the file, but it seems wrong

**Comment by wlach** [comment]
Ok, clearly something is broken with this test. I'll try to get that fixed soon along with some of the other nits you noticed. Thanks for the review!

File: tests/test_cli.py [hunk 3]
```diff
@@ -1,23 +1,413 @@
...
+    ###########################################################
+    # verify2
+    ###########################################################
+    (server_a_verify2_path, server_b_verify2_path) = [
+        os.path.join(path, "intermediate", "external", "verify2")
+        for path in (server_a_bucket_path, server_b_bucket_path)
+    ]
+    (server_a_verify2_output_filename, server_b_verify2_output_filename) = [
+        os.path.join(path, "data.ndjson")
+        for path in (server_a_verify1_path, server_b_verify1_path)
```
Comment (acmiyaguchi) [inline tests/test_cli.py]:
```suggestion
        for path in (server_a_verify2_path, server_b_verify2_path)
```

Comment (acmiyaguchi) [reply tests/test_cli.py]: This should fix the test so it prints out the correct aggregates.

**Comment by wlach** [comment]
Thanks for the help! We now actually test for the correct result at the end, in addition to all the nits/errors we noticed during review.

**Pull request merged**